File: env_variable/elixir_env_variable/env_variable.exs
#comment

defmodule Example do
  def env_variable_function(n) do
    n
  end
  def iterate(n) when n < 1 do
    #IO.puts "env_variable_function(" <> to_string(n) <> ")"
  end

  def iterate(n) do
    iterate(n - 1)
    IO.puts System.get_env("OS")
  end

  def main() do
    iterate(10)
  end
end

Example.main()

File: env_variable/lua_env_variable/env_variable.lua
-- comments
function env_variable_function (n)
  return n;
end

function main()
  for idx_i=1,10,1
  do
    print(os.getenv("OS"))
    print("env_variable_function(" .. idx_i .. ")=<" .. tostring(env_variable_function(idx_i)) .. ">")
  end
end

main()
